model/context.go

```diff
@@ -112,15 +113,20 @@ func DecodeContext(input interface{}, err error) (*Context, error) {
 	user, err := metadata.DecodeUser(userInp, err)
 	user = addUserAgent(user, http)
 
-	return &Context{
+	ctx := Context{
 		Http:    http,
 		Url:     url,
 		Labels:  labels,
 		Page:    page,
 		Custom:  custom,
 		User:    user,
 		Service: service,
-	}, err
+	}
+
+	if experimental, set := ctxInp["experimental"]; set {
```

Comment:
Does this only need to be set if exists / Can't this be moved into the literal above?